tests/bsim/bluetooth/host/l2cap/frags/prj.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_BT=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="L2CAP frags test"
+
+CONFIG_BT_EATT=n
+CONFIG_BT_L2CAP_ECRED=n
+
+CONFIG_BT_SMP=y # Next config depends on it
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+
+# Disable auto-initiated procedures so they don't
+# mess with the test's execution.
+CONFIG_BT_AUTO_PHY_UPDATE=n
+CONFIG_BT_AUTO_DATA_LEN_UPDATE=n
+CONFIG_BT_GAP_AUTO_UPDATE_CONN_PARAMS=n
+
+CONFIG_LOG=y
+CONFIG_ASSERT=y
+
+CONFIG_BT_L2CAP_LOG_LEVEL_DBG=y
+CONFIG_ARCH_POSIX_TRAP_ON_FATAL=y
```

Comment:
Should `CONFIG_ARCH_POSIX_TRAP_ON_FATAL` be enabled by default for BSIM tests?

Reply:
yes. the process will die anyway on assert.